src/web/client/WebExtensionContext.ts

```diff
@@ -783,6 +789,18 @@ class WebExtensionContext implements IWebExtensionContext {
     public async removeConnectedUserInContext(userId: string) {
         this.connectedUsers.removeUser(userId);
     }
+
+    public async openTeamsChat(userId: string): Promise<void> {
+        const mail = await this.graphClientService.getUserEmail(userId);
+        const teamsChatLink = getTeamChatURL(mail);
+        vscode.env.openExternal(vscode.Uri.parse(teamsChatLink.href));
+    }
+
+    public async openMail(userId: string): Promise<void> {
+        const mail = await this.graphClientService.getUserEmail(userId);
```

Comment:
handle case when mail is not retrieved - this.graphClientService.getUserEmail(userId)

Reply:
addressed